docs/index.md

```diff
@@ -3,41 +3,266 @@ hide:
   - navigation
 ---
 
-# Home
+
+# Welcome to Taipy Documentation!
 
 Taipy is an innovative **low-code** package to create complete applications.
-It is composed of two main independent components:
-**Taipy GUI** and **Taipy Core**.
+This documentation is divided in three main sections:
+
+- [**"Getting Started"**](getting_started/index.md) provides a step-by-step opening to Taipy. Taipy features are leveraged as the application becomes more and more complex.
+- [**"User Manual"**](manuals/about.md) describes what the product concepts are, and how you can use them.
+- [**"Reference Manual"**](manuals/reference) gathers all the Python APIs that Taipy exposes.
```

Comment:
Remove "Python" (there are REST and JavaScript APIs as well)

Reply:
Done